arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+                },
+                "requestId": {
+                    "type": "string",
+                    "format": "uuid",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "serverName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Server."
+                },
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Database."
+                },
+                "status": {
+                    "type": "string",
+                    "description": "Gets or sets the status message returned from the server."
+                },
+                "lastModifiedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation status last modified time."
+                },
+                "queuedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation queued time."
+                },
+                "blobUri": {
+                    "type": "string",
+                    "description": "Gets or sets the blob uri."
+                },
+                "errorMessage": {
+                    "type": "string",
+                    "description": "Gets or sets the error message returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportRequestParameters": {
+            "properties": {
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL database to import."
+                },
+                "edition": {
+                    "type": "string",
+                    "description": "Gets or sets the edition for the Azure Sql Database being created."
+                },
+                "serviceObjectiveName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the service objective to assign to the Azure SQL Database."
+                },
+                "maxSizeBytes": {
+                    "type": "string",
+                    "description": "Gets or sets the maximum size for the newly imported database."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/ExportRequestParameters"
+                }
+            ],
+            "description": "Import Azure SQL Database parameters."
+        },
+        "ExportRequestParameters": {
+            "properties": {
+                "storageKeyType": {
+                    "type": "string",
+                    "description": "Gets or sets the type of the storage key to use. Valid values are StorageAccessKey and SharedAccessKey."
```

Comment:
updated.

---
In reply to: [80153026](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80153026) [](ancestors = 80153026)